game/resources.cpp

```diff
@@ -144,6 +172,7 @@ void Resources::loadVdfs(const std::vector<std::u16string>& modvdfs, bool modFil
   }
 
 Resources::~Resources() {
+  DmLoader_release(dmLoader);
```

Comment:
Since the GothicVR team, who brought the issue of broken music up initially, want to use it in Unity, the API needs to be exposed as `extern "C"`. I just didn't feel like writing a C++ library and then having to deal with writing another C wrapper like I did for [ZenKit](https://github.com/GothicKit/ZenKitCAPI). Also, I just wanted to expand my C knowlegde :)